frontend/src/app/modules/common/form-field/form-field.component.html

```diff
@@ -1,24 +1,37 @@
 <label class="op-form-field--label-wrap">
```

Comment:
This will need a `for` if we're in non-wrapping mode

Reply:
Thank you for spotting that. I just realize that the only component that needs an 'inline' label, CkEditor, is not an 'input' and it seems that it doesn't have a label in OpenProject. I read that labels are just for inputs, what should we do?

Reply:
Maybe ignore it for now, until we actually need it? :D I don't think we need it for the ckeditor necessarily